vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -196,6 +196,20 @@ public String getPlaceholder() {
         return getPlaceholderString();
     }
 
+    @Override
+    public void setHelperText(String helperText) {
+        super.setHelperText(helperText);
+    }
+
+    /**
+     * String used for the helper text.
+     *
+     * @return the {@code helper-text} property from the webcomponent
```

Comment:
I do see that `webcomponent` is used without a space in the older Javadocs too (in unrelated methods), those could also be fixed to `web component` but not required for this PR since they're not all related to this issue.

Reply:
Fixed the typos, but didn't change the other occurrences of `webcomponent` since there are many and not really related with this issue.